components/library/webm/libwebm/Makefile

```diff
@@ -42,6 +42,7 @@ TARGET.32 = x86-solaris-gcc
 TARGET.64 = x86_64-solaris-gcc
 
 CXXFLAGS += -std=gnu++11
+LDFLAGS += -Wl,-R
```

Comment:
I think it does not... I just copied your suggestion. From the few samples
in repo, seems there should be optional(?) comma and some libpath. No idea
what to do or why, especially why it is mis-generated in the first place.

--
Typos courtesy of GMail on my Samsung Android
31.05.2016 17:37 пользователь "Alexander Pyhalov" <notifications@github.com>
написал:

> In components/library/webm/libwebm/Makefile
> <https://github.com/OpenIndiana/oi-userland/pull/2067#discussion_r65207153>
> :
>
> > @@ -42,6 +42,7 @@ TARGET.32 = x86-solaris-gcc
> >  TARGET.64 = x86_64-solaris-gcc
> >
> >  CXXFLAGS += -std=gnu++11
> > +LDFLAGS += -Wl,-R
>
> How does it help???
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/OpenIndiana/oi-userland/pull/2067/files/0fec5f9329b63b3f5aee75977b0ed4e1bc9aed4f..31a4570fd74ad6c7d6cde3aee8b796653cd69551#r65207153>,
> or mute the thread
> <https://github.com/notifications/unsubscribe/ABj5lCCGx8OsRur89OafGM6IVzFSvXt-ks5qHFWzgaJpZM4IpMZj>
> .
>